src/Core/Users/class.user.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface;
+
+Class User implements UserInterface {
+

```

Comment:
No class variables are declared. Please declare all variables that the class is using.